### PR TITLE
Added `Connectwise Automate Server` Role

### PR DESCRIPTION
**Pull request by wmmatt**

### CODE REVIEW

File: Function.Service-Check.ps1 [hunk 1]
```diff
@@ -107,6 +108,7 @@ Function Service-Check {
         'MSSQL'                             { [array]$ServiceList += 'MSSQLSERVER','SQLBrowser','SQLWriter','MsDtsServer100','MsDtsServer 110','MsDtsServer120','MsDtsServer130','MsDtsServer140','MSSQLServerOLAPService','SQLServerAgent' }
         'MySQL'                             { [array]$ServiceList += 'MySQL' }
         'Nessus'                            { [array]$ServiceList += 'Tenable Nessus' }
+        'Netwrix'                           { {array}$ServiceList += 'NwADASitSvc','NwAdfsSvc','NwArchiveSvc','NwCfgServerSvc','NwCoreSvc','NwDataClassificationSvc','NwDataCollectionCoreSvc','NwFileStorageSvc','NwManagementSvc','NwNetworkDeviceSvc','NwNLASvc','NwOracleSvc','NwSqlaHost','NwSyslogCollectionSvc','NwUBACoreSvc','NwUserActivitySvc','NwWatchdogSvc','NwWebAPISvc','NwWsaHostSvc' }
```
Comment (sethbrasile) [inline Function.Service-Check.ps1:111]: typo: accidentally used curlies instead of square brackets

Comment (sethbrasile) [reply Function.Service-Check.ps1:111]: fix incoming

Comment (sethbrasile) [reply Function.Service-Check.ps1]: fixed

**Comment by sethbrasile** [comment]
Fixed an easy issue, LGTM 👍🏻

**Review by sethbrasile — APPROVED** [review]

**Comment by wmmatt** [comment]
I forgot to add 'Netwrix' to the valid set where the roles are called so I just added that

**Comment by sethbrasile** [comment]
Good catch! I missed that!

**Pull request merged**